index.html

```diff
@@ -53,6 +48,171 @@ <h2>I am a Software Developer</h2>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-plceholder">
+                        <h4>Website Protfolio</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-profisional">
+                        <h4>Professional Art Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
+                <div class="work-card">
+                    <div class="card-availability">
+                        <h4>Data Dashboard  Healthcare</h4>
+                       
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div> 
+                <div class="work-card">
+                    <div class="card-plceholder">
+                        <h4>Website Protfolio</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div> 
+              
+            </div>
+        </section><!-- #Recent Works -->
```

Comment:
Please be sure that your design occupies all the width of the mobile screen. It's probably an issue with hard-coded widths or some issues with your margins. That is causing an empty space on the right of the screen

#### Your design 
![image](https://user-images.githubusercontent.com/15807343/149384216-958431dc-7d51-472e-b67e-ece41a218b28.png)